### ISSUE TITLE
weird blank setting on kOS difficulty screen

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#2781]
This blank toggle started showing up after one of my recent changes.

Find out what it is and fix it
![screenshot21](https://user-images.githubusercontent.com/5216848/96778164-dd208580-13b0-11eb-91ba-091ae615b74c.png)
.

**Comment by Dunbaratu** [comment]
Trial and error indicated it's this setting in KOSCustomParameters.cs, line 51:

```
        [GameParameters.CustomParameterUI("")]
        public bool passedClickThroughCheck = false;
```

But I don't know what causes that to appear on the difficulty screen when there's another setting just like it which doesn't, on line 45:

```
        [GameParameters.CustomParameterUI("")]
        public bool migrated = false;
```

**Comment by Dunbaratu** [comment]
Something changed in Clickthroughblocker v0.10.13 when a new button was added that kinda broke kOS - I have an issue there to go check on - fix this when that is answered.

**Comment by Dunbaratu** [comment]
Closed by #2797 but I typed the wrong issue number the first time so it failed to auto-close.
